### DISCUSSION TITLE
Mitigate browser caching when updating site

### DISCUSSION THREAD

**Discussion by adeturner** [facebook/docusaurus#9461] [Q&A] [answered]
Any pointers on how to prevent browser caching of a docusaurus app (2.4.3)?
For example after each build that adds new categories, the menu items do not update without a shift F5.

**Comment by slorber** [comment]
Hey

It's a single page application so you can't refresh Docusaurus by simply navigation, you have to reload the page

That's how SPAs work and is unrelated to browser caching. We don't control http cache headers set by your host.

**Reply by adeturner** [reply]
Thanks @slorber, its not happening on navigation. The container build/update was last night, and this is affecting brand new browser sessions today (which have accessed the site on prior days)

**Reply by adeturner** [reply]
Also it looks like it only affects the root page. If navigate to a url for a sub page the menu is correct
Even after F5, the root page reverts to the old one from disk cache

**Reply by slorber** [reply, marked as answer]
We don't control caching you setup yourself.

If your web server does not serve the latest assets, html/js/css, it's a configuration issue on your host, not a Docusaurus problem. 

Docusaurus is only responsible for building the static assets, not serving them.
